### ISSUE TITLE
Better local file spotting / Set local files' folder manually

### ISSUE THREAD

**Issue by Tvtossen** [z411/trackma#580]
I've had some bad experience with the local file detection (for instance reading Eva 1.11 as Eva episodes 1, and Aria the Animation as Hamatora THE ANIMATION, CardCaptor Sakura Clear-Hen OP 2 as Movie 2, list goes on)
It'd be great to either have more accurate library searching, being able to circumvent bad library searching via manually assigning a folder to a title, or both

**Comment by rakhmanovda** [comment]
I agree with @Tvtossen that manually assigning a folder should be an option. I have many anime not being found by Trackma. And sometimes people might have folder named in different languages
